### ISSUE TITLE
Support ordering by order of files in pane

### ISSUE THREAD

**Issue by talshorer** [oriolmirosa/open-files#12]
Love this package, but I would personally prefer if the files were ordered the same way they're ordered within the pane (so the leftmost tab in the pane is at the top of the list, and so forth).
Is such a thing possible?

**Comment by oriolmirosa** [comment]
You can modify the order of the files in the settings. I use the same system that the [tab-smart-sort](https://atom.io/packages/tab-smart-sort) package, which allows you to set the order of your tabs. So, if you install tab-smart-sort and set the same order of files in open-files (for instance, directory – extension – filename as 'dir, ext, base'), then you'll have what you want. I'm closing the issue, but let me know if I misunderstood and you can open it again.

**Comment by talshorer** [comment]
I think you misunderstood me, so let me try to explain again:
Currently I can reorder tabs around in panes freely, and I end up sorting them so I can move quickly between related files (e.g. a class and its main user might end up near each other despite being far alphabetically).
When you work on a large project and you end up with 20+ open files, remembering the exact order within panes becomes impossible. Some editors list the open files in the side bar by the order of tabs, and when tabs are moved the side bar is changed to reflect that. This way if I'm looking for a specific file all I really need to do is throw a look at the side bar, which is a lot easier than going through everything by hand.
In short, what I'm asking for is to keep sorting tabs within panes by hand, but to have the side bar reflect that order.

**Comment by oriolmirosa** [comment]
Thanks for the clarification, @talshorer. I now see what you meant. I think that this is related to #7, and I definitely need to come up with a flexible approach that can take care of all these use cases. I'm moving across the country this weekend and I need to find a new job and get married in the next month, so I may not be able to get to this right away, but I hope I'll be able to work on this soon.

**Comment by vvs** [comment]
Let me add my +1 to this feature request. I am also arranging my open files manually, depending on the task at hand. Most important ones are typically at the beginning of the tab bar.

It would be really, really nice for the open file list and the tab bar to be in sync w.r.t. manual ordering.

Ideally, yeah, when file order is changed in one place, it should be reflected in the other. Personally, I could live with just one-way sync, when changing the tab order, the open files list would just reflect that.

Thank you for the awesome plugin and good luck with our relocation and marriage!

**Comment by jpromocion** [comment]
Interesting option
